portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,4 +41,13 @@ private String convertToJsonUsingGson(ArrayList<String> lst) {
     String json = gson.toJson(lst);
     return json;
   }
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
```

Comment:
Please try to make the comment more useful.  In this case, this comment doesn't add much value.  It's obvious from the next line that we are reading text-input, likely a form.

So we could either skip the comment to keep the code clean.  If needed, we the comment must add value. Such as explaining the format of text-input, or where this input is set etc.